source/code/plugins/filter_mysql_logs.rb

```diff
@@ -0,0 +1,31 @@
+module Fluent
+  class MysqlLogsFilter < Filter
+    Fluent::Plugin.register_filter('filter_mysql_logs', self)
+    
+    def initialize
+      super
+      require 'socket'
+    end
+
+    def configure(conf)
+      super
+    end
+
+    def start
+      super
+    end
+
+    def shutdown
+      super
+    end
+
+    def filter(tag, time, record)
+      record["ResourceName"] = 'Mysql'
+      record["Computer"] = IPSocket.getaddress(Socket.gethostname)
```

Comment:
use OMS::Common.get_hostname to get the Computer name to be consistent with the other data.